docs/docs/audit-with-lighthouse.md

```diff
@@ -1,51 +1,51 @@
 ---
-title: Audit with Lighthouse
+title: Auditar con Lighthouse
 ---
 
-Quoting from the [Lighthouse website](https://developers.google.com/web/tools/lighthouse/):
+Citando el [sitio web de Lighthouse](https://developers.google.com/web/tools/lighthouse/):
 
-> Lighthouse is an open-source, automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps (PWAs), and more.
+> Lighthouse es una herramienta automatizada de código abierto diseñada para mejorar la calidad de tus apps web. Puedes ejecutarla en cualquier página web, pública o que requiera autenticación. Tiene auditorías de rendimiento, accesibilidad, aplicaciones web progresivas (_PWAs_), y más.
```

Comment:
En este caso la traducción de `the quality of web pages` es `la calidad de las páginas webs` en lugar de **la calidad de tus apps web**